builtin/rebase--interactive2.c

```diff
@@ -70,12 +70,13 @@ int cmd_rebase__interactive(int argc, const char **argv, const char *prefix)
 	struct replay_opts opts = REPLAY_OPTS_INIT;
 	unsigned flags = 0, keep_empty = 0, rebase_merges = 0, autosquash = 0;
 	int abbreviate_commands = 0, rebase_cousins = -1, ret;
-	const char *head_hash = NULL, *onto = NULL, *restrict_revision = NULL,
+	const char *onto = NULL, *restrict_revision = NULL,
 		*squash_onto = NULL, *upstream = NULL, *head_name = NULL,
 		*switch_to = NULL, *cmd = NULL, *onto_name = NULL;
-	char *raw_strategies = NULL, *revisions = NULL, *shortrevisions = NULL;
-	struct argv_array make_script_args = ARGV_ARRAY_INIT;
-	FILE *todo_list;
+	char *raw_strategies = NULL;
+	enum {
+		CONTINUE = 1, SKIP, EDIT_TODO, SHOW_CURRENT_PATCH
+	} command = 0;
```

Comment:
Maybe the enum could also have NONE=0, like:

+	enum {
+		NONE=0, CONTINUE, SKIP, EDIT_TODO, SHOW_CURRENT_PATCH
+	} command = 0;